bindings/python/test/guidance_law/test_effectivity_provider.py

```
@@ -0,0 +1,199 @@
+        self._score = score
+
+    def is_effective(self, state, direction, thrust_acceleration):
+        return self._gate
+
+    def compute_score(self, state, direction, thrust_acceleration):
+        return self._score
+
+
+class TestComposite:
+    def test_constructor_rejects_empty(self):
+        with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)